homeworks/taridaniel/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,8 @@
+name = "Daniel Tari"
```

Comment:
Ezeket a változókat a felhasználótól kérd be az input() function-nel, és hívd meg rájuk a .strip() és a .capitalize() metódusokat, hogy biztosítsd hogy nincs fölösleges szóköz, és nagybetűvel kezdődik a név!